.github/workflows/preview-release.yml

```diff
@@ -21,6 +21,12 @@ jobs:
       - name: Give GitHub some time to process the deletion, otherwise our release shows up as draft. Sigh.
         run: sleep 10
 
+      - name: Extract changelog section for release
```

Comment:
Took me a while to understand that https://github.com/thomaseizinger/xmr-btc-swap/releases/tag/preview is actually correctly displaying nothing in the body, because you obviously triggered a test release (0.4.0), which updates the changelog and the `unreleased` section is empty - this the `preview` release will have an empty body, which is correct :)

Reply:
Ah yes, might be worth special case that at some point :)